packages/core/src/actors.ts

```diff
@@ -2,30 +2,18 @@ import type {
   InvokeCallback,
```

Comment:
All of this is in the `actors.ts` but we refer to them as behaviors. It feels like we should do something about it because it feels like we are getting lost ourselves in the used nomenclature

Reply:
I'll do this in a separate PR (want to rename to `behaviors.ts` because it's a noisy change